snippets/javascript.json

```
@@ -653,5 +653,868 @@
     "prefix": [
       "callout3DLargeProps"
     ]
+  },
+  "Add CSV Layer": {
+    "body": [
+    "CSVLayer({",
+    "\turl: \"${1:https://earthquake.usgs.gov/earthquakes/feed/v1.0/summary/2.5_week.csv}\",",
+    "\tcopyright: \"${2:USGS Earthquakes}\",",
+    "\tpopupTemplate: ${3:template}",
+    "});",
+    "map.add(csvLayer);"
+    ],
+    "description": "Add a CSVLayer with a URL",
```

```suggestion
    "description": "Add a CSVLayer (point layer) based on a CSV file (.csv, .txt) with a URL. CSV is a plain-text file format used to represent tabular data, including geographic point features (latitude, longitude). AMD path: esri/layers/CSVLayer | ESM path: @arcgis/core/layers/CSVLayer.js",
```
Following `description` [conventions](https://github.com/Esri/arcgis-js-vscode-snippets/blob/master/CONTRIBUTING.md#description)